kompose.spec

```diff
@@ -52,7 +52,7 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        1.1.0
+Version:        1.2.0
 Release:        0.1%{?dist}
```

Comment:
@surajssd Can you make this release as 1, this is just a bump version which we usually used when a patch required. I am not sure why we have it as `0.1`?

Reply:
done!